docs/pages/access-controls/access-graph/aws-sync.mdx

```diff
@@ -0,0 +1,219 @@
+---
+title: Discover AWS access patterns with Teleport Access Graph
+description: Describes how to import and visualize AWS accounts access patterns using Teleport Access Graph.
+---
+
+The Teleport Access Graph offers invaluable insights into access patterns within
+your AWS account. By scanning IAM permissions, users, groups, resources, and 
+identities, it provides a visual representation and aids in enhancing the 
+permission model within your AWS environment. This functionality enables you
+to address queries such as:
+
+- What resources are accessible to AWS users and roles?
+- Which resources can be reached via identities associated with EC2 instances?
+- What AWS resources can Teleport users access when connecting to EC2 nodes?
+
+Utilizing the Access Graph to analyze IAM permissions within an AWS
+account necessitates the setup of the Teleport Access Graph (TAG)
+service, a Discovery Service, and integration with your AWS account.
+
+Teleport Access Graph is a feature of the [Teleport
+Policy](https://goteleport.com/platform/policy/) product that is only available
+to Teleport Enterprise customers.
+
+After logging into the Teleport UI, go to the Management tab. If enabled, 
+Access Graph options can be found under the Permission Management section.
+
+## How AWS Sync Process works
+
+Teleport Access Graph seamlessly synchronizes various AWS resources,
+including IAM Policies, Groups, Users, User Groups, EC2 instances, 
+EKS clusters, and RDS databases. These resources are then visualized
+using the graph representation detailed in the
+[Access Graph page](../access-graph.mdx).
+
+The importing process involves two primary steps:
+
+### Polling Cloud APIs
+
+The Teleport Discovery Service plays a pivotal role in this process
+by continuously scanning the configured AWS accounts. 
+At intervals of 15 minutes, it retrieves the following resources from your
+AWS account:
+
+- AWS Users
+- AWS Groups
+- AWS User Groups
+- AWS IAM Roles
+- AWS IAM Policies
+- AWS EKS Clusters
+- AWS RDS Databases
+- AWS S3 Buckets
+
+Once all the necessary resources are fetched, the Teleport Discovery
+Service pushes them to the Teleport Access Graph (TAG) service, 
+ensuring that the Access Graph remains updated with the latest
+information from your AWS environment.
+
+### Importing resources
+
+Teleport Access Graph delves into the IAM policies, identities,
+and resources retrieved from your AWS account, crafting a 
+graphical representation thereof.
+
+Throughout this procedure, Access Graph seamlessly aligns
```

Comment:
This sentence sounds like a sales pitch. I don't think it provides any value to someone looking to understand how things work or how to set it up.

Reply:
Removed it.